token/cli/src/output.rs

```diff
@@ -176,6 +187,9 @@ impl fmt::Display for CliMultisig {
 #[serde(rename_all = "camelCase")]
 pub(crate) struct CliTokenAccount {
     pub(crate) address: String,
+    pub(crate) program_id: String,
+    pub(crate) epoch: u64,
```

Comment:
i was split on whether to include `epoch` here, because we only need it for mint, but getting it for account is trivial and passing an `Option<u64>` into `display_ui_extension` feels silly

Reply:
My take is to not have the `epoch` at all and just show both transfer fees and their epoch.  This way you can easily spot an upcoming rug pull, or see how it's been increased

Reply:
right, showing both does make sense. i think i still want it so i can display which is active and whether the other is upcoming or historical tho

Reply:
Mm true... I'll leave the final choice to you

Reply:
Sorry for the fly-by, but I think it will be confusing for consumers to see `epoch` anywhere. If you need it for extension display, can you add the `#[serde(skip_serializing)]` tag so the field doesn't appear in json output?

Reply:
thank you! i forgot that these structures were dumped as json and had been using them as if they were purely internal